force-app/main/default/classes/SOQL.cls

```diff
@@ -53,6 +61,16 @@ public with sharing class SOQL {
     //[public | private ] [static] data_type method_name (input parameters) {
     //  The body of the method
     //}
+    public static List<Id> GetContactIdsByLastName() {
+        List<Contact> conWithId = [SELECT Id, Name FROM Contact WHERE LastName = 'Smith' OR LastName = 'Jones'];
```

Comment:
Take a look at the IN operator if you need to check if a Field has any one of a specific set of values, in this case Last Names.